package.json

```diff
@@ -12,7 +12,7 @@
   "main": "dist/index.js",
   "scripts": {
     "start": "docz dev",
-    "build": "rm -rf dist && tsc --project tsconfig-build.json && cpx './src/**/!(docAssets)/*.css' 'dist'",
+    "build": "rm -rf dist && tsc --project tsconfig-build.json && cpx './src/**/!(docAssets)/*.css' 'dist' && cpx './src/*.css' 'dist'",
```

Comment:
The build command is getting long. Can we make a `copy:assets` script and use that in the `build` script?